cobaya/samplers/polychord/polychord.py

```diff
@@ -69,7 +69,7 @@ def initialize(self):
         self.nDerived = (len(self.model.parameterization.derived_params()) +
                          len(self.model.prior) + len(self.model.likelihood._likelihoods))
         if self.logzero is None:
-            self.logzero = np.nan_to_num(-np.inf)
+            self.logzero = -1e300
```

Comment:
The particular choice of fixed value doesn't matter that much (MontePython uses `-1e30`, I believe). So if anyone prefers a different fixed value, feel free to change.